pkg/controller/utils/elasticsearch.go

```diff
@@ -98,15 +102,15 @@ func ElasticsearchSecrets(ctx context.Context, userSecretNames []string, cli cli
 	return append(esUserSecrets, esCertSecret), nil
 }
 
-// GetElasticsearchClusterConfig retrieves the config map containing the elasticsearch configuration values, such as the
+// Getrelasticsearch.ClusterConfig retrieves the config map containing the elasticsearch configuration values, such as the
```

Comment:
I think this function should stay the same name, or be moved into an ES specific package, otherwise it's too vague for a "utils" package. 

Also the comment seems a little wonky now?

Reply:
Ya I did some global search and replace, I missed a few of these.